src/verilator/nvdla/NVDLA.py

```diff
@@ -33,7 +33,8 @@
 class NVDLAWrapper(DrivenObject):
   type = 'NVDLAWrapper'
   cxx_header = "verilator/nvdla/nvdla_wrapper.hh"
-  do_trace = Param.Bool(True, "Use the specified trace file?")
   buf_clear_cycles = Param.Int(4096,
     "Number of cycles to clear hardware buffers")
-  trace_file = Param.String("input.txn", "Name of trace file to run test" )
+  nvdla_sys = Param.System(Parent.any, "Pointer to nvdla system")
+  axi_2_gem5 = Param.MemObject(Parent.any,
```

Comment:
Will this really be a parent? I suggest not using a proxy here.

Reply:
Yeah I wasn't really sure what would go here. based off the Proxy Factory code I figured any would go down the tree and eventually hit what I was asking for. Would the system i passed have a way of getting the memobject? I couldn't find any getters in the class declaration

Reply:
No.. you would probably want to do the following in the config file:

```
system.nvdla = NVDLAWrapper()
system.nvdla.axi_2_gem5 = AXIWrapper()
```
I'm not exactly sure what the `AXIWrapper` is here or if you have that done already. But basically, you don't think you want the `nvdla` to be a child of it's memory "port". Instead you want the memory port to be a child of the `nvdla`.

Reply:
I see. I think I fixed it.